.github/workflows/main.yml

```
@@ -12,9 +12,9 @@ jobs:
     name: Main Process
     runs-on: ubuntu-latest
     env:
-      GO_VERSION: 1.19
-      GOLANGCI_LINT_VERSION: v1.50.0
-      YAEGI_VERSION: v0.14.2
+      GO_VERSION: 1.23
+      GOLANGCI_LINT_VERSION: v1.61.1
```

```suggestion
      GOLANGCI_LINT_VERSION: v1.61.0
```

golangci-lint v1.61.1 doesn't exist :wink:

---

whoops sorry!